### PR TITLE
Add BRC project.

### PR DESCRIPTION
**Pull request by willfurnass**
@annakrystalli @bobturneruk This is a rough first draft and I've likely missed some key contributions!

### CODE REVIEW

**Comment by bobturneruk** [comment]
Great stuff @willfurnass - I'd consider being less specific about mostract (given the status of the project), and adding more about the healthcare benefits of the work. But time is always short!

**Review by bobturneruk — APPROVED** [review]

**Comment by willfurnass** [comment]
Ta @bobturneruk.  Re health benefits: would you say the main thing would be to stress the value per data point and therefore importance of data mgmt/governance?

**Review by annakrystalli — APPROVED** [review]

**Comment by bobturneruk** [comment]
@willfurnass I was thinking more like (potentially) improvements to patient quality of life, more accurate prognosis, quicker assessment of whether a treatment is working, reduced NHS costs.

File: _project_descriptions/brc.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  - Project officers/administrators at STH
+  - The Scientific Computing team at STH
+  - This RSE team
+
+RSE time has been costed for the duration of the BRC project (at 50% FTE),
+with the main study we're involved with being [MoStrAct][mostract].
+MoStrAct aims to explore whether data from movement sensors such as gait monitors can
+provide biomarkers for changes in neurological conditions,
+with additional sources of information such as standard clinical tests and MRI imaging data
+being used to provide independent information on changes.
```
Comment (willfurnass) [inline _project_descriptions/brc.md:22]:
```suggestion
being used to provide independent information on changes.
Reliable gait-based biomarkers have the potential to make the monitoring of chronic neurological disorders a cheaper and more continual process.
```

File: _project_descriptions/brc.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  - Storing study-specific gait sensor data.
+
+Other RSE contributions to the project include:
+
+  - Defining and documenting data dictionaries (schemas)
+  - Developing and supporting the development of software tooling for processing and visualising gait data (using MATLAB and R)
+  - Providing training to researchers on tools and methods that should
+    make the computational aspects of the workflows more robust and reproducible
+    (such as training in software version control using Git and GitHub).
+
```
Comment (willfurnass) [inline _project_descriptions/brc.md:39]:
```suggestion

In general, many contributions have been around data storage/management/governance and making analysis of that data more robust/reproducible, with this focus resulting in part from the high high cost (in terms of time and money) per data point in this study.
```

**Comment by willfurnass** [comment]
> @willfurnass I was thinking more like (potentially) improvements to patient quality of life, more accurate prognosis, quicker assessment of whether a treatment is working, reduced NHS costs.

Added a little to address this.

**Pull request merged**